src/smexperiments/trial.py

```diff
@@ -153,6 +156,7 @@ def list(
             api_types.TrialSummary.from_boto,
             "TrialSummaries",
             experiment_name=experiment_name,
+            trial_component_name=trial_component_name,
```

Comment:
does this depend on a new boto version? please verify min boto version specified in dependencies has this change

Reply:
from https://github.com/boto/boto3/blob/2a4efd21345597189fbe3cccaa4b5593da504cb1/.changes/1.10.20.json looks like last update on sagemaker client is happening in 1.10.20 version

Reply:
actually mini boto3 version is https://boto3.amazonaws.com/v1/documentation/api/1.12.8/reference/services/sagemaker.html#SageMaker.Client.list_trials

Reply:
looks like min boto version needs to be bumped then 
https://github.com/aws/sagemaker-experiments/blob/master/setup.py#L27

Reply:
sure, updated